From: Thomas Renninger <trenn@suse.de>
Subject: Fix display of idle boxes
Patch-Mainline: not yet
References: none

This fix is needed for eye of gnome and firefox svn viewers.
Inkscape can also handle the broken case.

Signed-off-by: Thomas Renninger <trenn@suse.de>

---
 tools/perf/util/svghelper.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: linux-2.6.37-master/tools/perf/util/svghelper.c
===================================================================
--- linux-2.6.37-master.orig/tools/perf/util/svghelper.c
+++ linux-2.6.37-master/tools/perf/util/svghelper.c
@@ -455,9 +455,9 @@ void svg_legenda(void)
 		return;
 
 	svg_legenda_box(0,	"Running", "sample");
-	svg_legenda_box(100,	"Idle","rect.c1");
-	svg_legenda_box(200,	"Deeper Idle", "rect.c3");
-	svg_legenda_box(350,	"Deepest Idle", "rect.c6");
+	svg_legenda_box(100,	"Idle","c1");
+	svg_legenda_box(200,	"Deeper Idle", "c3");
+	svg_legenda_box(350,	"Deepest Idle", "c6");
 	svg_legenda_box(550,	"Sleeping", "process2");
 	svg_legenda_box(650,	"Waiting for cpu", "waiting");
 	svg_legenda_box(800,	"Blocked on IO", "blocked");
